backend/src/test/java/codezap/member/repository/MemberJpaRepositoryTest.java

```diff
@@ -1,45 +1,139 @@
+import codezap.category.repository.CategoryRepository;
 import codezap.fixture.CategoryFixture;
+import codezap.fixture.MemberFixture;
+import codezap.global.exception.CodeZapException;
 import codezap.global.repository.JpaRepositoryTest;
 import codezap.member.domain.Member;
-import codezap.member.fixture.MemberFixture;
 import codezap.template.domain.Template;
 import codezap.template.repository.TemplateJpaRepository;
+import codezap.template.repository.TemplateRepository;
 
 @JpaRepositoryTest
 class MemberJpaRepositoryTest {
 
-    private final MemberJpaRepository memberJpaRepository;
-    private final TemplateJpaRepository templateJpaRepository;
-    private final CategoryJpaRepository categoryJpaRepository;
+    private final MemberRepository memberRepository;
+    private final TemplateRepository templateRepository;
+    private final CategoryRepository categoryRepository;
 
     @Autowired
-    MemberJpaRepositoryTest(MemberJpaRepository memberJpaRepository, TemplateJpaRepository templateJpaRepository,
-            CategoryJpaRepository categoryJpaRepository
+    MemberJpaRepositoryTest(
+            MemberJpaRepository memberRepository,
+            TemplateJpaRepository templateRepository,
+            CategoryJpaRepository categoryRepository
     ) {
-        this.memberJpaRepository = memberJpaRepository;
-        this.templateJpaRepository = templateJpaRepository;
-        this.categoryJpaRepository = categoryJpaRepository;
+        this.memberRepository = memberRepository;
+        this.templateRepository = templateRepository;
+        this.categoryRepository = categoryRepository;
     }
 
-    @Test
-    void fetchByTemplateId() {
-        // Then
-        Member member = ((JpaRepository<Member, Long>)memberJpaRepository).save(MemberFixture.memberFixture());
-        Category category = ((JpaRepository<Category, Long>)categoryJpaRepository).save(CategoryFixture.getFirstCategory());
-        Template template = new Template(member, "title", "description", category);
-        Template savedTemplate = ((JpaRepository<Template, Long>)templateJpaRepository).save(template);
+    @Nested
+    @DisplayName("id로 Member 조회")
+    class fetchById {
+        @Test
+        @DisplayName("성공 : id로 Member를 알아낼 수 있다.")
+        void fetchByIdSuccess() {
+            Member member = MemberFixture.getFirstMember();
+            member = memberRepository.save(member);
 
-        assertThat(memberJpaRepository.fetchByTemplateId(savedTemplate.getId())).isEqualTo(member);
+            Member actual = memberRepository.fetchById(member.getId());
+
+            assertThat(actual).isEqualTo(member);
+        }
+
+        @Test
+        @DisplayName("실패 : 존재하지 않는 id인 경우 에러가 발생한다.")
+        void fetchByIdFailByNotExistsId() {
+            long id = 100;
+
+            assertThatThrownBy(() -> memberRepository.fetchById(id))
+                    .isInstanceOf(CodeZapException.class)
+                    .hasMessage("식별자 " + id + "에 해당하는 멤버가 존재하지 않습니다.");
```

Comment:
```suggestion
            long notExistId = 100;

            assertThatThrownBy(() -> memberRepository.fetchById(notExistId))
                    .isInstanceOf(CodeZapException.class)
                    .hasMessage("식별자 " + notExistId + "에 해당하는 멤버가 존재하지 않습니다.");
```
[제안]
변수 명을 조금 더 직관적으로 바꿔보는 것은 어떨까요?